src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.io.IOException;
+
+/**
+ * Duke class, main driver of the duke application. The duke app is an
+ * application that takes users inputs and stores tasks in a list as a
+ * reminder for the user.
+ */
+public class Duke {
+    private TaskList taskList;
+    private Ui ui;
```

Comment:
The variable `ui` does not appear to be used. Perhaps you should delete it if you don't need it anymore.